crates/autopilot/src/solvable_orders.rs

```diff
@@ -1,6 +1,9 @@
 use {
     crate::{
-        domain::{self},
+        domain::{
+            eth,
+            {self},
```

Comment:
intersting that clippy doesn't complain about the braces here.

Reply:
Good catch!